### PR TITLE
[tests-only][full-ci]Fix failing expiration dates tests

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR fixes the expiration-related tests that were failing because it's almost end of the year and the `selecting this year 2021` button is disabled in the UI. The PR check if the month of the expiration date is `December` directly day of the expiration is selected and this fixes the issue

## Related Issue

- Fixes : https://github.com/owncloud/web/issues/6071

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature [hunk 1]
```diff
@@ -11,7 +11,7 @@ Feature: Versions of a file
       | user0    |
       | Alice    |
 
-  @disablePreviews @skipOnOC10 @issue-5853
+  @disablePreviews  @skipOnOC10 @issue-5853
```
Comment (Talank) [inline tests/acceptance/features/webUIFilesActionMenu/versions.feature:14]: May be this change is not necessary
```suggestion
  @disablePreviews @skipOnOC10 @issue-5853
```

File: tests/acceptance/pageObjects/FilesPageElement/expirationDatePicker.js [hunk 2]
```diff
@@ -98,7 +98,9 @@ module.exports = {
      */
     isExpiryDateDisabled: async function (pastDate) {
       let disabled = false
-
+      if (pastDate.getMonth() === 11) {
```
Comment (phil-davis) [inline tests/acceptance/pageObjects/FilesPageElement/expirationDatePicker.js:101]: `11` is "magic". At first I thought it is some hard-coded reference to November (the 11th month). Now I have read the PR and it seems to be the zero-based index of December (the 12th month).

Please add a comment. Or put this in a constant somewhere (with a comment) and use the constant here and at line 169.

**Comment by phil-davis** [comment]
I rebased - somehow the branch of this PR was 11 commits behind master. Let's see what CI says now.

**Comment by phil-davis** [comment]
A smoke test is still failing. https://drone.owncloud.com/owncloud/web/20563/10/16
I thought that uses Playwright and should be completely unrelated to any Nightwatch test code???

**Comment by phil-davis** [comment]
But those smoke tests passed in the nightly last night: https://drone.owncloud.com/owncloud/web/20564/8/16
What is going on?

**Comment by phil-davis** [comment]
Smoke test failed against oC10 https://drone.owncloud.com/owncloud/web/20565/10/16
But passed against oCIS.
I will restart again. We need to see if it can pass sometimes.

File: tests/acceptance/pageObjects/FilesPageElement/expirationDatePicker.js (hunk 2 above)
Comment (SwikritiT) [reply tests/acceptance/pageObjects/FilesPageElement/expirationDatePicker.js]: I changed the function, it should be easier to understand now.

**Comment by SwikritiT** [comment]
The smoke test passed in this recent build https://drone.owncloud.com/owncloud/web/20569/10/16. The rest of the build should also pass but let's see what happems

**Review by phil-davis — APPROVED** [review]
Let's get this merged to get CI  green. Then I can report separately about the smoke test intermittent fails.

**Pull request merged**